app/src/components/v-form/form-field-interface.vue

```diff
@@ -27,6 +27,7 @@
 			:primary-key="primaryKey"
 			:length="field.schema && field.schema.max_length"
 			@input="$emit('update:modelValue', $event)"
+			@set-other="$emit('update:otherValue', $event)"
```

Comment:
Lets rename `set-other` to `set-field-value`, and use that everywhere.

The event `update:otherValue` could just be `setFieldValue`, as it doesn't have any two-way binding with the parent form

Reply:
Fixed by the last commit : set-field-value and setFieldValue are used instead of otherValue.